lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -81,6 +82,12 @@ protected function configure() : void
                 InputOption::VALUE_OPTIONAL,
                 'Check Database Platform to the generated code.',
                 true
+            )
+            ->addOption(
+                'allow-no-diff',
+                null,
+                InputOption::VALUE_NONE,
+                "Don't throw an exception when no changes detected (CI)."
```

Comment:
I'd avoid abbreviations, or specifying context (this option might be helpful outside CI, in a pre-push hook for instance).
```suggestion
                "Do not throw an exception when no changes are detected."
```

Reply:
Well, to create this option I used another option as reference. The option in question is `allow-no-migration` from `MigrateCommand`, as you can see the option's description is:

https://github.com/doctrine/migrations/blob/0069f40df5d7fbe54c32f03843bb36a1c7123bc9/lib/Doctrine/Migrations/Tools/Console/Command/MigrateCommand.php#L62

The effect of this option is the same as the options I have written: display a message instead of throwing an exception. I have no problem changing the option's description, but I suggest to also change the message for `allow-no-migration` option, to keep some coherence.

Reply:
👍 please also change the `allow-no-migration` option then.

Reply:
Done.